### DISCUSSION TITLE
Some repositores are not getting indexed in cs.github.com

### DISCUSSION THREAD

**Discussion by r0ny123** [community/community#10457] [Other Feature Feedback, Questions, & Ideas] [answered]
I searched for `repo:g-les/100DaysofYARA` on cs.github.com but it threw an error `Unknown repo: "g-les/100DaysofYARA"`, `Query is not satisfiable`. Is it bug or intentional?

![image](https://user-images.githubusercontent.com/49360849/150684013-e1b7c4a1-89ea-4118-a9d7-a4d1f5b4f1fc.png)

**Comment by look** [comment, marked as answer]
Sorry, at this time not all repositories are indexed. We're indexing about 5 million public repositories plus the content from users in the technical preview.

We're working on increasing the number of repositories indexed, so hopefully we can index this one soon!

**Comment by r0ny123** [comment]
Thanks @look. can't wait to get them indexed! :)
